packages/preferences/src/browser/abstract-resource-preference-provider.ts

```diff
@@ -53,13 +53,7 @@ export abstract class AbstractResourcePreferenceProvider implements PreferencePr
     protected async readPreferences(): Promise<void> {
         const newContent = await this.readContents();
         const strippedContent = jsoncparser.stripComments(newContent);
-        const errors: jsoncparser.ParseError[] = [];
-        this.preferences = jsoncparser.parse(strippedContent, errors);
-        if (errors.length) {
-            for (const error of errors) {
-                this.logger.error("JSON parsing error", error);
-            }
-        }
+        this.preferences = jsoncparser.parse(strippedContent);
```

Comment:
It pollutes the output log on each character in settings.json, mostly it is meaningless since a user can have some broken state.

Reply:
but should `onDidPreferencesChanged` be fired in case of parse errors? the doc says, it will return a (partial) result. that could interfere with default value semantics, right?

Reply:
I don't think my change changed it.

As for now if settings.json is broken we will fire at least parseable values. If we don't fire default always be used. It also does not seem to be correct. Please file an issue if you think that something is fishy here :)

Reply:
will do. 
you're right, that didn't change it.